AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java

```diff
@@ -299,6 +300,7 @@ static void setModifiedBigPictureStyle(Notification.Builder notificationBuilder,
    * @param messageText String with text for push notification.
    * @return Notification.BigPictureStyle or null.
    */
+  @TargetApi(16)
```

Comment:
Should these be changed from 16-26 as well?

Reply:
Yeah. I'm putting the lowest needed TargetApi numbers in here. 
Here's why
1. The gradle compilation process checks to ensure that we aren't using any android calls that are below the minimum supported version (14 in this case)
2. However, in certain methods, if we are handling the different API calls ourselves (through branching etc), then we can let the compiler know that and ask it for a higher API check.
3. In this case, I'm setting the TargetApi wrt the branching logic in the function, 16 in most cases, but 26 for notification channels.